prj.conf

```diff
@@ -18,17 +29,18 @@ CONFIG_IMG_MANAGER=y
 CONFIG_IMG_ERASE_PROGRESSIVELY=y
 CONFIG_REBOOT=y
 
-CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
-
-CONFIG_JSON_LIBRARY=y
-
+# The rest of the runtime credentials config
+CONFIG_GOLIOTH_SAMPLE_HARDCODED_CREDENTIALS=n
 CONFIG_SETTINGS=y
-CONFIG_GOLIOTH_SETTINGS=y
 CONFIG_SETTINGS_RUNTIME=y
 CONFIG_GOLIOTH_SAMPLE_PSK_SETTINGS=y
 CONFIG_GOLIOTH_SAMPLE_SETTINGS_AUTOLOAD=y
 CONFIG_GOLIOTH_SAMPLE_SETTINGS_SHELL=y
 
-CONFIG_GOLIOTH_RPC=y
+# Misc.
+CONFIG_JSON_LIBRARY=y
 CONFIG_NETWORK_INFO=y
 CONFIG_I2C=y
+
+# Firmware version used in DFU process
+CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION="1.99.0"
```

Comment:
Doesn't need to gate this PR, but it would be helpful to walk through an example of how you would like the version numbers to increment in the `main` branch over multiple releases (e.g. immediately after the `2.0.0` release, does the version stay at `2.0.0` in the `main` branch until the next release? or does it get immediately bumped to `2.99.0`? etc.)

Reply:
That's a good point. For now my goal is to ensure the version number of the release is reflected here. That's what will be automatically built by CI and we should be directing our users to checkout releases and not build on commits that come after. I don't see a reason to rev this number other than at release time.